src/seam-connect/routes.ts

```diff
@@ -98,13 +98,16 @@ export abstract class Routes {
 
   private async createActionAttemptAndWait<T extends ActionType>(
     request: AxiosRequestConfig
-  ): Promise<ActionAttemptResultTypeMap[T]> {
+  ): Promise<ActionAttemptResultTypeMap[T] & { action_attempt_id: string }> {
     const pendingActionAttempt = await this.makeRequestAndFormat<
       ActionAttemptCreateResponse<T>
     >("action_attempt", request)
     const actionAttempt = await this.awaitActionAttempt<T>(pendingActionAttempt)
 
-    return actionAttempt.result
+    return {
+      ...(actionAttempt.result as any),
+      action_attempt_id: actionAttempt.action_attempt_id,
```

Comment:
we could also include the entire action attempt

Reply:
it's a bit verbose

Reply:
hmm yeah might as well